assets/src/edit-story/app/api/apiProvider.js

```diff
@@ -141,14 +141,16 @@ function APIProvider( { children } ) {
 	 * Upload file to via REST API.
 	 *
 	 * @param {File}    file           Media File to Save.
+	 * @param {?Object} additionalData Additional data to include in the request.
 	 *
 	 * @return {Promise} Media Object Promise.
 	 */
 	const uploadMedia = useCallback(
-		( file ) => {
+		( file, additionalData ) => {
 			// Create upload payload
 			const data = new window.FormData();
 			data.append( 'file', file, file.name || file.type.replace( '/', '.' ) );
+			forEach( additionalData, ( value, key ) => data.append( key, value ) );
```

Comment:
Why use lodash instead of a simple `for` loop or `Array.prototype.forEach()`?

Reply:
It is not a array here, is an object.

Reply:
So `for ... in` should work: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in

Or even a combination with `Object.keys`

Reply:
So 
```
for ( const [ value, key  ] in additionalData ){
    data.append( key, value );
}
```

Reply:
Fixed in https://github.com/google/web-stories-wp/pull/195/commits/1d1aa2e3e2cbb98b417b75bb7cf35f54f64be44a